server/src/testing/client.rs

```diff
@@ -317,14 +356,22 @@ impl Client {
         Ok(())
     }
 
-    fn spawn_stack_poller(&self) {
-        let net = self.net.clone();
-        tokio::task::spawn_local(async move {
-            loop {
-                tokio::time::delay_for(STACK_POLL_INTERVAL).await;
-                net.poll();
+    async fn resolve_node(&self, node_id: NodeId, addr: SocketAddr) -> anyhow::Result<VirtNode> {
+        let ip = to_ipv6(node_id);
+        match self.get_node(&ip.octets()).await {
+            Some(node) => Ok(node),
+            None => match async {
+                let session = self.session(addr).await?;
```

Comment:
Resolving node should always use central server.
I will fix this in p2p PR

Reply:
Yes, I would like to use other nodes too. But in this case we are asking about Node info. If we already have session with someone, we know his node info. If we don't have session, we can't ask him about this

Reply:
The arguments are not related. `node_id` identifies the node we need a descriptor of, whereas `addr` identifies a session ("server") we're going to use for the lookup. The latter may be either the central relay session address or the address of a node we are currently using as a relay